eclass/qt5-build.eclass

```diff
@@ -67,6 +83,58 @@ SLOT=5/$(ver_cut 1-2)
 QT5_MINOR_VERSION=$(ver_cut 2)
 readonly QT5_MINOR_VERSION
 
+case ${PV} in
+	5.12.4) QT5_DOCDATE=201906140200 ;;
+	5.13.0) QT5_DOCDATE=201906171600 ;;
+esac
+
+if [[ -n ${QT5_DOCDATE} ]]; then
+	QT5_DOCURI="https://download.qt.io/online/qtsdkrepository/linux_x64/desktop/qt5_$(ver_rs 1-3 '')_src_doc_examples/qt.qt5.$(ver_rs 1-3 '').doc/"
+	case ${PN} in
+		designer) : ${QT5_DOCTARS="qt${PN} qtuitools"} ;;
```

Comment:
uhm... the idea was to have these declaration in each individual ebuild (with a default of `${PN}`) similar to the `QT5_MODULE` variable

Reply:
sure, it's a list I had to assemble anyway and something I can easily put into every ebuild instead. ebuilds could set `QT5_DOCS` which is default `false` in eclass, set it to `true` to have the eclass map it to `${PN}`, and for any other value, use its content as list of tarball names.